website/content/docs/configuration/entropy-augmentation.mdx

```diff
@@ -59,3 +59,5 @@ These parameters apply to the `entropy` stanza in the Vault configuration file:
 - `mode` `(string: <required>)`: The mode determines which Vault operations requiring
   entropy will sample entropy from the external source. Currently, the only mode supported
   is `augmentation` which sources entropy for [Critical Security Parameters (CSPs)](/vault/docs/enterprise/entropy-augmentation#critical-security-parameters-csps).
+- `seal_name` `(string: '')`: Specifies which seal (by name) in a   [Seal HA](/vault/docs/concepts/seal#seal-high-availability) setup to use to  source entropy. 
+  By default, Vault sources entropy from the first available seal by increasing priority.
```

Comment:
```suggestion
  By default, Vault sources entropy from the first available seal by increasing its priority.
```

Would this make sense? It reads better for me, but it's your call. I'm leaving an approval anyways 😄

Reply:
Alternative suggestion to hopefully address some of the confusion:
```suggestion
  By default, Vault sources entropy from the first available seal moving from lowest to highest priority.
```